src/redux/rockets/rocketReduce.js

```diff
@@ -0,0 +1,49 @@
+const FETCH_ROCKETS = 'FETCH_ROCKETS';
+const ADD_RESERVATION = 'ADD_RESERVATION';
+const CANCEL_RESERVATION = 'CANCEL_RESERVATION';
+const URL = 'https://api.spacexdata.com/v3/rockets';
+
+export const fetchRockets = async (dispatch) => {
+  const response = await fetch(URL);
+  const data = await response.json();
+  dispatch({ type: FETCH_ROCKETS, payload: data });
+};
```

Comment:
- [ ] Similar to the mission fetch, after fetching the rockets you are expected to filter the data to avoid saving unnecessary data in the store. You are required to only save the following data as listed in the requirements: `id, rocket_name, description, flickr_images`. But you are saving all data from the API fetch. See image

  Rockets state | Missions state
  :---:|:---:
  ![rocket-1](https://user-images.githubusercontent.com/81468675/187947320-c223d046-8ae1-489f-a789-1e6cad89401f.jpg) | ![mission](https://user-images.githubusercontent.com/81468675/187950122-0d712566-417c-489a-9432-179081b9e238.jpg)